src/App.js

```diff
@@ -22,20 +22,21 @@ LogBox.ignoreLogs([
 ]);
 
 const App = () => (
-    <ComposeProviders
-        components={[
-            OnyxProvider,
-            SafeAreaProvider,
-            SafeArea,
-            LocaleContextProvider,
-            HTMLEngineProvider,
-        ]}
-    >
+    <ErrorBoundary errorMessage="NewExpensify crash caught by error boundary">
         <CustomStatusBar />
-        <ErrorBoundary errorMessage="NewExpensify crash caught by error boundary">
+
+        <ComposeProviders
```

Comment:
As an error could happen on context provider initialization, `ErrorBoundary` should wrap them to hide the splash screen is something goes wrong.